src/app/roles.tsx

```diff
@@ -1,43 +1,44 @@
 "use client";
 
-import { DatabaseEnums } from "@/databaseUtils";
+import { DatabaseRoles } from "@/databaseUtils";
 import React, { useState, createContext } from "react";
 
 interface Props {
     children: React.ReactNode;
 }
 
-type Roles = DatabaseEnums["role"] | "";
-
-type RolesInterface = {
-    [key in Roles]: string[];
-};
+export type Roles = DatabaseRoles;
```

Comment:
If we're just redeining `DabaseRoles`, can we just use `DatabaseRoles` and not `Roles` at all?

Reply:
Yes